.travis.yml

```diff
@@ -22,6 +22,7 @@ env:
   - CXX=g++-4.8 GCLOUD_TRACE_NEW_CONTEXT=1
 
 script:
+  - if [ $TRAVIS_NODE_VERSION -ge 8 ]; then yarn list; fi
```

Comment:
Why is this needed?

Reply:
We don't print package structure for Node 8 tests so I wanted to add that for debuggability -- but I realized that we don't for any Node version tests.

I'll remove the conditional in an upcoming PR... don't want to wait for CI to pass again before landing this change.